src/eval/execute.rs

```diff
@@ -95,6 +95,14 @@ fn generate_code_to_send(code: &str, bare: bool) -> String {
             code = body
         ))
     };
+
+    // Normalize the code if `raw` is false.
```

Comment:
I think the code is reasonably self-explanatory, so this comment is probably not very useful.